README.md

```diff
@@ -119,7 +119,7 @@ These are the following dependencies used to verify the testcases. Torch-TensorR
 - Libtorch 2.3.0 (built with CUDA 12.1)
 - CUDA 12.1
 - cuDNN 8.9.5
```

Comment:
Can you remove this line now as we are not explicity using cudnn as dependency ?